packages/zoe/src/zoeService/makeInvitation.js

```diff
@@ -1,13 +1,15 @@
 // @ts-check
 import { assert, details as X } from '@agoric/assert';
-import { AmountMath, makeIssuerKit, AssetKind } from '@agoric/ertp';
+import { AmountMath, makeDurableIssuerKit, AssetKind } from '@agoric/ertp';
 import { InvitationElementShape } from '../typeGuards.js';
 
 /**
+ * @param {import('@agoric/vat-data').Baggage} baggage
  * @param {ShutdownWithFailure | undefined} shutdownZoeVat
  */
-export const createInvitationKit = (shutdownZoeVat = undefined) => {
-  const invitationKit = makeIssuerKit(
+export const createInvitationKit = (baggage, shutdownZoeVat = undefined) => {
```

Comment:
What do you think of a naming convention where functions that 
   * take a baggage first argument,
   * call a `vivifySomething`
   * therefore need to be called during the first crank, if they might be revivifying

should themselves be called `vivifySomething`?

```suggestion
export const vivifyInvitationKit = (baggage, shutdownZoeVat = undefined) => {
```

Reply:
sounds like a good rule.